sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -426,6 +426,37 @@ RUN apt-get update && apt-get install -y eatmydata && eatmydata apt-get install
         default-jdk \
         libgoogle-gson-java
 
+{%- if CONFIGURED_PLATFORM != "marvell" %}
```

Comment:
is it better to use this one?

{%- if CONFIGURED_ARCH == "armhf" and CROSS_BUILD_ENVIRON != "y" %}

Reply:
I don't understand your suggestion. Will `CONFIGURED_ARCH == "armhf"` match vs platform?

Reply:
Done, @lguohan  Please check it again